cmd/github.go

```diff
@@ -72,14 +74,23 @@ func githubAppAccessTokenGenerateCommand(cmd *cobra.Command, args []string) {
 	if cfg.GitHub.AppID == "" {
 		exitWithUsage(cmd, "Missing app ID flag")
 	}
-	if cfg.GitHub.AppKey == "" {
-		exitWithUsage(cmd, "Missing app key flag")
+	if cfg.GitHub.AppKey == "" && cfg.GitHub.AppKeyFile == "" {
+		exitWithUsage(cmd, "Missing app key GPC_GITHUB_APP_KEY or key file flag")
 	}
 	if cfg.GitHub.Org == "" {
 		exitWithUsage(cmd, "Missing GitHub org flag")
 	}
 
-	token, err := gh.GenerateAppToken(cfg.GitHub.Org, cfg.GitHub.AppID, cfg.GitHub.AppKey)
+	appKey := cfg.GitHub.AppKey
+	if cfg.GitHub.AppKeyFile != "" {
+		buf, err := os.ReadFile(cfg.GitHub.AppKeyFile)
+		appKey = string(buf)
+		if err != nil {
+			logger.Fatal("ERROR: couldn't read key file: %w", err)
```

Comment:
Please change `%w` to `%s`. The `%w` format verb only works when using [fmt.Errorf](https://pkg.go.dev/fmt#Errorf) to create a wrapped error.
Using `%w` with the logger prints the following.
```
ERROR: couldn't read key file: %!w(*fs.PathError=&{open foo 2})
```

```suggestion
			logger.Fatal("ERROR: couldn't read key file: %s", err)
```